.github/workflows/deploy-squid-stage.yaml

```
@@ -1,121 +1,31 @@
 # naming convention: <project>/<environment>/<task>
-name: multix/stage/deploy
+name: subsquid aquarium deployment
 
 on:
   push:
     paths:
-      - 'packages/squid/**'
+      - 'squid/**'
       - '.github/workflows/deploy-squid-stage.yaml'
     branches:
       - main
-      # - ci/* # run this workflow if the branch starts with ci/*
-
-env:
-  # secrets are listed here to simplify the readability
-  AWS_REGION: '${{ secrets.AWS_REGION }}'
-  AWS_ACCOUNT_ID: '${{ secrets.AWS_ACCOUNT_ID }}'
-  AWS_ROLE_NAME: '${{ secrets.AWS_ROLE_NAME }}'
-  APP_STAGE: 'stage'
-  DOCKER_REGISTRY: 'ghcr.io'
-  DOCKER_TAG: 'latest'
-  ENV_FILE: '${{ secrets.ENV_FILE }}'
+      - tbaut-use-aquarium # run this workflow if the branch starts with ci/*
```

comment doesn't match value

---

right, I should remove this line alltogether before merging. Will change the comment in the mean time
```suggestion
      - tbaut-use-aquarium # run this workflow if the branch starts is tbaut-use-aquarium
```